Traducir.Web.Net/wwwroot/js/site.js

```diff
@@ -0,0 +1,4 @@
+﻿// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
```

Comment:
bootstrapped js?

Reply:
Sorry what do you mean?

Reply:
that it was added by dotnet automatically :) and we'll clean it up

Reply:
Ah ok! Since we're using a library named Bootstrap I was highly confused 😅